manifests/Quazar Conundrum.yaml

```diff
@@ -0,0 +1,11 @@
+name: The Quazar Condundrom
+authors: Anerix(Paradox)CorminQuanadol
+homepage: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/tree/main
+license: CC-BY-SA-4.0
+version: V0.1.5
```

Comment:
If you want to use autoupdate, this will need to be the same as your tag name (currently `Major_ReleaseV015`)

Reply:
oh god do i need to resubmit the PR? because i cant figure out how to edit the YAML from here

Reply:
alright this section is fixed, it should link directly to the release with version